snapcraft.yaml

```diff
@@ -1358,7 +1358,7 @@ parts:
     organize:
       lib/python3.12/site-packages/: lib/python3/dist-packages/
     prime:
-      - bin/uefivars
+      - bin/uefivars.py
```

Comment:
dont we need an organize statement o rename bin/uefivars to bin/uefivars.py?

Reply:
Yes, thanks.

I've build the snap and ensured that `prime/bin/uefivars.py` is in `prime/bin` directory.